src/detector.ts

```diff
@@ -225,7 +227,7 @@ function getEvent(type: EventType, node: HTMLElement): ProductEvent {
     t: Date.now(),
     page: getPage(),
     id: generateId(),
-    uid: getUserId(),
+    uid: window.TS.getUserId() || getUserId(),
```

Comment:
```suggestion
    uid: window.TS?.getUserId() ?? getUserId(),
```

Reply:
on second thought: we might wanna revert this or tell typescript to polyfill it for compatibility reasons?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

They seem to be available everywhere

Reply:
Lets clarify supported browsers in the README then.